docs/setup/upgrade/resolving-migration-failures.asciidoc

```diff
@@ -133,7 +133,7 @@ Migrations will fail if saved objects belong to an unknown
 saved object type. Unknown saved objects are typically caused by performing manual modifications
 to the {es} index (no longer allowed in 8.x), or by disabling a plugin that had previously created a saved object.
 
-We recommend using the {kibana-ref-all}/7.17/upgrade-assistant.html[Upgrade Assistant]
+We recommend using the {kibana-ref-all}/{version}/upgrade-assistant.html[Upgrade Assistant]
```

Comment:
I added a commit that revives the upgrade-assistant page, therefore you should be able to link to it like this:
```suggestion
We recommend using the <<upgrade-assistant,Upgrade Assistant>>
```